File: fixtures/sample-repo/src/go/auth/token.go
package auth

import (
	"fmt"
	"strings"
)

func IssueToken(user string) string {
	salt := helperSalt()
	return fmt.Sprintf("%s:%s", user, salt)
}

func helperSalt() string {
	return strings.ToLower("S3CRET")
}

File: fixtures/sample-repo/src/rust/token.rs
use std::collections::HashMap;

mod helpers;

pub fn issue_token(user: &str) -> String {
    let salt = helper_salt();
    format!("{user}:{salt}")
}

fn helper_salt() -> String {
    "s3cret".into()
}

pub async fn refresh_token(token: &str) -> String {
    issue_token(token)
}
